tests/ann/test_weaviate.py

```diff
@@ -237,3 +257,68 @@ def test_index_exists(embeddings, weaviate_client):
 def test_normalize_cosine_distance():
     assert ann.normalize_cosine_distance(0.0) == 1.0
     assert ann.normalize_cosine_distance(2.0) == -1.0
+
+
+def test_upsert(embeddings, weaviate_client):
```

Comment:
@davidmezzetti this this passes

Reply:
If you are reusing the config, you would need to clear the `offset` field or set it to the appropriate value.